pandas/core/indexing.py

```diff
@@ -935,7 +938,11 @@ def _getitem_lowerdim(self, tup: tuple):
             #  is equivalent.
             #  (see the other place where we call _handle_lowerdim_multi_index_axis0)
             with suppress(IndexingError):
-                return self._handle_lowerdim_multi_index_axis0(tup)
+                # error "_LocationIndexer" has no attribute
+                # "_handle_lowerdim_multi_index_axis0"
```

Comment:
I was about to make _LocationIndexer an abstract class but a sub-class (I think _iLoc...) does not implement `_handle_lowerdim_multi_index_axis0` even though the class is being instantiated.